### PR TITLE
feat: changed some colors of the dark theme

### PR DESCRIPTION
**Pull request by MayukhSamadder2000**
Screenshot 2023-02-23 at 11 37 10 PM

### CODE REVIEW

File: src/components/SwitchInput/SwitchInput.tsx [hunk 1]
```diff
@@ -14,11 +14,13 @@ const Label = styled.div`
...
 
-const Wrapper = styled.div<{ inline?: boolean; disabled: boolean; }>`
+const Wrapper = styled.div<{ inline?: boolean; disabled: boolean }>`
   margin-bottom: 18px;
   width: 100%;
 
-  ${({ inline }) => inline && `
+  ${({ inline }) =>
+    inline &&
+    `
```
Comment (poocart) [inline src/components/SwitchInput/SwitchInput.tsx:23]: was ok
```suggestion
  ${({ inline }) => inline && `
```

File: src/components/SwitchInput/SwitchInput.tsx [hunk 2]
```diff
@@ -29,8 +31,10 @@ const Wrapper = styled.div<{ inline?: boolean; disabled: boolean; }>`
       margin-right: 8px;
     }
   `}
-  
-  ${({ disabled }) => disabled && `
+
+  ${({ disabled }) =>
+    disabled &&
+    `
```
Comment (poocart) [inline src/components/SwitchInput/SwitchInput.tsx:37]: was ok
```suggestion
  ${({ disabled }) => disabled && `
```

File: src/components/SwitchInput/SwitchInput.tsx [hunk 3]
```diff
@@ -56,15 +60,16 @@ const SwitchOption = styled.div<{ isActive: boolean; disabled: boolean; percenta
   min-height: 34px;
   line-height: 34px;
 
-  ${({ isActive, disabled }) => !isActive && !disabled && `
+  ${({ isActive, disabled }) =>
+    !isActive &&
+    !disabled &&
+    `
```
Comment (poocart) [inline src/components/SwitchInput/SwitchInput.tsx:66]: was ok
```suggestion
  ${({ isActive, disabled }) => !isActive &&  !disabled && `
```

File: src/components/SwitchInput/SwitchInput.tsx [hunk 4]
```diff
@@ -56,15 +60,16 @@ const SwitchOption = styled.div<{ isActive: boolean; disabled: boolean; percenta
...
-
-    &:hover {
-      text-decoration: underline;
-    }
   `}
 
-  ${({ isActive, theme }) => isActive && `
+  ${({ isActive, theme }) =>
+    isActive &&
+    `
```
Comment (poocart) [inline src/components/SwitchInput/SwitchInput.tsx:72]: was ok
```suggestion
  ${({ isActive, theme }) =>  isActive && `
```

File: src/providers/TransactionBuilderContextProvider.tsx [hunk 5]
```diff
@@ -80,14 +80,6 @@ const TransactionBlockListItemWrapper = styled.div<{ disabled?: boolean }>`
   &:last-child {
     margin-bottom: 0;
   }
-
-  ${({ disabled }) =>
```
Comment (poocart) [inline src/providers/TransactionBuilderContextProvider.tsx:84]: no longer any indication for disabled?

File: src/providers/TransactionBuilderContextProvider.tsx [hunk 6]
```diff
@@ -80,14 +80,6 @@ const TransactionBlockListItemWrapper = styled.div<{ disabled?: boolean }>`
...
-    !disabled &&
-    `
-    &:hover {
```
Comment (poocart) [inline src/providers/TransactionBuilderContextProvider.tsx:87]: no longer any indication for hover?

File: src/components/Button/SecondaryButton.tsx [hunk 7]
```diff
@@ -6,18 +6,14 @@ const SecondaryButton = styled.button<{
...
+  font-family: 'PTRootUIWebMedium', sans-serif;
   border: none;
   background: transparent;
   font-size: 16px;
   padding: 0;
   ${({ marginTop }) => marginTop && `margin-top: ${marginTop}px`};
   color: ${({ color, theme }) => color || theme.color.text.buttonSecondary};
   cursor: pointer;
-
-  &:hover {
```
Comment (poocart) [inline src/components/Button/SecondaryButton.tsx:18]: no longer any indication for hover?

**Review by poocart — APPROVED** [review]
minor comments for unnecessary changes, hope it can stay as it was

File: src/utils/theme.ts [hunk 8]
```diff
@@ -162,27 +162,27 @@ export const darkTheme: Theme = {
...
+      switchInputActiveTab:
+        'linear-gradient(to bottom, #734fb3, #422d66), linear-gradient(to bottom, #3d265c, #222130)',
       switchInputInactiveTab: 'transparent',
       pill: '#2b2640',
       checkboxInputInactive: '#665c99',
       dropdownHoverColor: '#443d66',
       selectInputExpandedHover: '#443d66',
-      walletButton: 'linear-gradient(to bottom, #6154a3, #292246)',
-      walletChainDropdown: '#1a1726',
-      walletChainButtonActive: '#443d66',
```
Comment (Yash-Chaubal) [inline src/utils/theme.ts:185]: Any reason we're removing the dark theme versions of the wallet styles?

File: src/utils/theme.ts [hunk 9]
```diff
@@ -195,20 +195,19 @@ export const darkTheme: Theme = {
...
-      textInputSecondary: '#ffeee6',
+      textInputSecondary: '#9466e6',
       switchInputActiveTab: '#ffeee6',
-      switchInputInactiveTab: '#bbb8cc',
+      switchInputInactiveTab: '#9466e6',
       selectInputImagePlaceholder: '#ffeee6',
       cardDisabled: '#605e5e',
       pill: '#bbb8cc',
       pillValue: '#ffeee6',
-      walletDropdownIcon: '#ff884d',
```
Comment (Yash-Chaubal) [inline src/utils/theme.ts:211]: Same as above

**Review by Yash-Chaubal — APPROVED** [review]
Look's good, but just a question about some removed colours

File: src/components/Button/SecondaryButton.tsx (hunk 7 above)
Comment (MayukhSamadder2000) [reply src/components/Button/SecondaryButton.tsx:18]: Yeah it was mentioned in the ticket, to remove underline on hover

File: src/utils/theme.ts (hunk 8 above)
Comment (MayukhSamadder2000) [reply src/utils/theme.ts:185]: Hmm, good catch here, I'll see this, probably it got removed while migrating the styles from the buidler landing page.

File: src/providers/TransactionBuilderContextProvider.tsx (hunk 6 above)
Comment (MayukhSamadder2000) [reply src/providers/TransactionBuilderContextProvider.tsx:87]: Yeah it was mentioned in the ticket, to remove underline on hover

File: src/components/SwitchInput/SwitchInput.tsx (hunk 4 above)
Comment (MayukhSamadder2000) [reply src/components/SwitchInput/SwitchInput.tsx:72]: Auto format did it, no Idea how it got turned on, I'll revert this.

**Pull request merged**